ui/components/multichain/asset-picker-amount/asset-picker-modal/asset-picker-modal-tabs.tsx

```diff
@@ -32,6 +32,7 @@ export const AssetPickerModalTabs = ({
       <Tabs
         defaultActiveTabKey={defaultActiveTabKey}
         tabsClassName="modal-tab__tabs"
+        onTabClick={() => null}
```

Comment:
Tried a few things here like setting defaults in the javascript component but wasn't able to fix this issue short of refactoring the `Tabs` component. The @MetaMask/design-system team is working on Tabs this quarter so this is a temporary fix until the [design-system-react](https://github.com/MetaMask/metamask-design-system/tree/a1d04ad2e1c226912d8fe2586cb7a1836a085f62/packages/design-system-react) component is ready